packages/core/package.json

```diff
@@ -34,7 +34,7 @@
   "scripts": {
     "lib:ses": "cp ../../node_modules/ses/dist/lockdown.umd.js ./lib/lockdown.umd.js",
     "lint:deps": "depcheck",
-    "test": "ava && npm run test:ses",
+    "test": "npm run test:ses && ava",
```

Comment:
Why?

Reply:
so that you can do  `npm test test/filename` like in every other package